stories/Components/Header.tsx

```diff
@@ -42,12 +41,15 @@ export const Header = ({ user, onLogin, onLogout, onCreateAccount }: HeaderProps
             <span className="welcome">
               Welcome, <b>{user.name}</b>!
             </span>
-            <Button size="small" onClick={onLogout} label="Log out" />
+            {/* <Button onClick={onLogout} label="Log out" /> */}
+            <Button  label="Log out" />
           </>
         ) : (
           <>
-            <Button size="small" onClick={onLogin} label="Log in" />
-            <Button primary size="small" onClick={onCreateAccount} label="Sign up" />
+            {/* <Button onClick={onLogin} label="Log in" />
+            <Button onClick={onCreateAccount} label="Sign up" /> */}
```

Comment:
Remove these entirely?

Reply:
I left them since they were outside the scope of my ticket. Previously mentioned here was a `CustomButton` component that I have no idea where it came from.

Reply:
Mike is correct. There's no need to leave unused code. It's in your commit history. Remove it to have a clean file.